<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-G8O6OG0" failures="0" tests="1" name="com.TestNGDemos.D06DataDrivenTesting" time="0.000" errors="1" timestamp="2024-06-21T20:09:45 IST" skipped="0">
  <testcase classname="com.TestNGDemos.D06DataDrivenTesting" name="loginToOHRM" time="0.000">
    <error message="
Method public void com.TestNGDemos.D06DataDrivenTesting.loginToOHRM(java.lang.String,java.lang.String) requires a @DataProvider named : getloginData" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method public void com.TestNGDemos.D06DataDrivenTesting.loginToOHRM(java.lang.String,java.lang.String) requires a @DataProvider named : getloginData
at org.testng.internal.Parameters.findDataProvider(Parameters.java:532)
at org.testng.internal.Parameters.handleParameters(Parameters.java:769)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- loginToOHRM -->
</testsuite> <!-- com.TestNGDemos.D06DataDrivenTesting -->
